File: Rest-Api-V3/market-data-public/depth/depth.py
import requests

def fetch_market_depth(market_code, level):
    url = f'https://api.ox.fun/v3/depth?marketCode={market_code}&level={level}'

    try:
        response = requests.get(url)
        response.raise_for_status()  # Raise an error for bad responses
        print('Market Depth Data:', response.json())
    except requests.exceptions.RequestException as error:
        print('Error fetching market depth data:', error)

# Example usage
fetch_market_depth('BTC-USD-SWAP-LIN', 10)

File: Rest-Api-V3/market-data-public/depth/depth.js
const axios = require('axios');

async function fetchMarketDepth(marketCode, level) {
  const url = `https://api.ox.fun/v3/depth?marketCode=${marketCode}&level=${level}`;

  try {
    const response = await axios.get(url);
    console.log('Market Depth Data:', response.data);
  } catch (error) {
    console.error('Error fetching market depth data:', error.response ? error.response.data : error.message);
  }
}

// Example usage
fetchMarketDepth('BTC-USD-SWAP-LIN', 10);